fleet/lib/ingress-nginx/fleet.yaml

```diff
@@ -10,6 +12,17 @@ helm:
   values:
     controller:
       kind: DaemonSet
+      metrics:
+        enabled: true
+        serviceMonitor:
+          enabled: true
+          additionalLabels:
+            lsst.io/monitor: "true"
+        # TODO: handle error: cannot unmarshal rule from spec
```

Comment:
It sounds like this would break the deployment in other clusters?  If so, this should either be changed to a targetCustomization or split out into its own PR.

Reply:
The TODO applies to the lines lower. The `prometheusRules` included on the chart can't be unmarshalled by the current Prometheus operator format. I'll adjust the TODO to be more clear and correct.